demos/999-storage-import-vhd-blob/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,239 @@
+{
```

Comment:
this needs to be simplified... you can use any of the following VHDs available in the test subscription:

  "GEN-SPECIALIZED-WINVHD-URI"
  "GEN-GENERALIZED-WINVHD-URI"
  "GEN-GENERALIZED-UBUNTUVHD-URI"

And then you only need a storageAccount for the destination.

Reply:
done